notes/views/new_funct.py

```
@@ -0,0 +1,28 @@
+STORY_SHARE_MESSAGE_SUFFIX = "hello"
+
+def get_qs_data_list(qs, data_key):
+    qs = []
+    if data_key:
+        qs = qs[0]
+    return qs
+
+def get_story_text(story_info):
+    ds = get_qs_data_list(story_info, data=story_info)
+    if ds:
+        story_share_text = story_info.get('story_title') + "\n" + STORY_SHARE_MESSAGE_SUFFIX
```

Ruff

Local variable `story_share_text` is assigned to but never used

Remove assignment to unused variable `story_share_text`

(F841)